frontend/gfx/ingame.js

```diff
@@ -203,16 +203,16 @@ function setGameState(e) {
 function ppUpdate(e) {
   const typeDiv = e.type === 'Baron' ? baron : elder
   const div = typeDiv.querySelector('.pp-bar')
-  const timer = typeDiv.querySelector('.timer')
+  // const timer = typeDiv.querySelector('.timer')
```

Comment:
timmer should not me commented out, this si a bug and was fixed again with the newest change

Reply:
Ohh sorry. I just looked at the main branch and saw that they were. Thanks!